docs/start/requirements.md

```diff
@@ -19,15 +19,44 @@ huronOS works as a live USB, so, in your requirements, you will need to consider
 
 ## Computer Requirements:
-- Network: We highly recommend wired networking with DHCP. WiFi is also supported, but we are not able to automate authentication yet.
+- **Network:** We highly recommend wired networking with DHCP. WiFi is also supported, but we are not able to automate authentication yet.
 
-- RAM: We recommend **at least** 8 GiB of RAM, 16+ GiB is preferred. Consider that huronOS lives on RAM and the disk is the RAM as well, which is then synched to the USB drive for persistence. Your RAM is your drive limit and it will be used for the system files + persistence + RAM + caches, and there's no swap. So, to avoid memory shortage issues, we do recommend having at least 8 GiB. With 4 GiB you'll likely run out of memory with a bad programmed DP. 
+- **RAM:** We recommend **at least** 8 GiB of RAM, 16+ GiB is preferred. Consider that huronOS lives on RAM and the disk is the RAM as well, which is then synched to the USB drive for persistence. Your RAM is your drive limit and it will be used for the system files + persistence + RAM + caches, and there's no swap. So, to avoid memory shortage issues, we do recommend having at least 8 GiB. With 4 GiB you'll likely run out of memory with a bad programmed DP. 
 
-- Disk: huronOS does not use the computer's disk drive.
+- **Disk:** huronOS does not use the computer's disk drive.
 
-- Processor: We only support x86_64 processors, when more human resources are available, we might explore releasing also an x86_32 and ARM build.
+- **Processor:** We only support x86_64 processors, when more human resources are available, we might explore releasing also an x86_32 and ARM build.
+
+- **GPU:** Integrated is more than enough.
+
+## Network Requirements
+- **DHCP for IPv4**  
+    DHCP can improve the setup times for contest arenas using huronOS. Currently, huronOS cannot persistently save static IP configurations, so the setup needs to be done manually each time the system is booted. This is why it is highly encouraged to use DHCP.
+
+- **Disable IPv6**  
+    huronOS is currently not capable of supporting IPv6 automated firewall setup. Due to this limitation, it is recommended to disable IPv6 on your network and use only IPv4. Having a dual IP stack for v4 and v6 can lead to IPv4 firewall not working properly, as users might have access to IPv6-allowed sites.
+
+- **Wired Network**  
+    While WiFi is supported, a wired network is always preferred over a wireless network for a more reliable and automated setup. Additionally, wireless cards come from a variety of vendors, and the drivers might not be available with the huronOS ISO image. We strive to provide as many drivers as possible, but sometimes you will need to install them manually. With Ethernet, this isn't the case as most generic drivers work on all types of Ethernet interfaces.
```

Comment:
Just to avoid over reiteration: "...is always prefered over a wireless one for a more...".
And this is optional but I think it works better: "With Ethernet, this isn't an issue as most..."